assets/src/design-system/components/contextMenu/animatedContextMenu.js

```diff
@@ -270,14 +277,24 @@ function AnimationContainer({ children, isOpen, ...props }) {
 }
 AnimationContainer.propTypes = {
   isOpen: PropTypes.bool,
+  isReady: PropTypes.bool,
+  onAnimationComplete: PropTypes.func.isRequired,
   children: PropTypes.node,
 };
 
-const AnimatedContextMenu = ({ isOpen, items, ...props }) => (
-  <AnimationContainer isOpen={isOpen}>
-    <Menu items={items} isOpen={isOpen} {...props} />
-  </AnimationContainer>
-);
+const AnimatedContextMenu = ({ isOpen, items, ...props }) => {
+  const [isReady, setIsReady] = useState(false);
+
+  return (
+    <AnimationContainer
+      isReady={isReady}
+      onAnimationComplete={setIsReady}
+      isOpen={isOpen}
+    >
+      <Menu items={items} isOpen={isOpen && isReady} {...props} />
+    </AnimationContainer>
```

Comment:
Chatted with @BrittanyIRL about this - looks like this should fix the bug and allow us to keep the focus of the first clickable item when the menu opens!

The menu needed to know when the animation was complete since the effect was running before the menu was finished animating

Reply:
Oops no. for some reason having the `useEffect` in the menu doesn't allow the click events to fire 🤔. For now let's just remove it